source/clear-linux/tutorials/gce/gce.rst

```diff
@@ -0,0 +1,122 @@
+.. _clear-on-gce:
+
+Launch |CL-ATTR| Compute Engine on Google Cloud Platform
+########################################################
+
+|CL-ATTR| is a Linux distribution designed for bare mental and virtual machines
+running from the cloud to the edge. This tutorial walks you through the steps
+to create virtual machines in
+`Google Cloud Platform (GCP) <https://cloud.google.com/>`_ running |CL|.
+Reference :ref:`azure` and :ref:`aws-web` for launching |CL| VMs on
+Microsoft\* Azure\* cloud services and Amazon\* Web Services respectively.
+
+#. Google Cloud Platform uses **Projects** to manage resources. Sign in
+   your Google account on the Google Cloud Console, and select or create
+   a new project.
+   Reference the
+   `Quickstart Using a Linux VM <https://cloud.google.com/compute/docs/quickstart-linux>`_
+   guide to get familiar with the process of creating VM instances in GCP.
+
+   .. figure:: figures/00-sign-in.png
+      :scale: 50 %
+      :alt: Sign in Google services
+
+#. Download the |CL| image archive for Google Compute Engine (GCE)
+   *clear-<release number>-gce.tar.gz*
+   from the |CL-PRJ| image repository. You don't need to uncompress the image
+   archive, the intact file will be uploaded to the Google Cloud Storage later.
+
+#. Click the *Navigation menu* icon on the upper left screen menu, select
+   the *Storage* item from the side bar on the left, and then click the
+   ``CREATE BUCKET`` button on the Cloud Storage page to create a *bucket*
+   for hosting the |CL| image source archive downloaded in the previous step.
+   The bucket name must be unique as buckets in the Cloud Storage share
+   a single global namespace.
+
+   .. figure:: figures/01-cloud-storage.png
```

Comment:
For all the browser screenshots I suggest only capturing the website window and crop out the browser from the view. Web browsers  update frequently and are customizeable so the look and feel just adds noise.

Also it hides the URL which can contain revealing information,

Reply:
@puneetse thanks for feedbacks, your considerations are valid, I'll address all of them.